src/prefect/cli/agent.py

```diff
@@ -195,6 +195,7 @@ async def start(
                     printer=app.console.print,
                     run_once=run_once,
                     jitter_range=0.3,
+                    backoff=4,  # Up to ~1 minute interval during backoff
```

Comment:
Should the backoff get added to workers's loops as well?

Reply:
We definitely should. It could be added to this PR or a different PR. It'll be a small change either way.